css/quiz.css

```diff
@@ -0,0 +1,126 @@
+/***************************
+GENERAL STYLING
+****************************/
+
+
+
+body {
+	margin: 0;
+	font-size: 1em;
+	font-weight: normal;
+	font-family: sans-serif;
+	color: black;
+	background-color: lightgray;
+}
+
+
+/***************************
+NAVIGATION
+****************************/
+
+
+#nav_bar {
```

Comment:
Adding this id isn't really necessary. The semantic `<nav>` tag keeps you from having to use a nearly-redundant id.